application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -69,61 +69,64 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String originalMessageId = event.getTarget().getId();
         String originalChannelId = event.getChannel().getId();
         String authorId = event.getTarget().getAuthor().getId();
+        String mostCommonTag = defaultTags.get(0);
 
-        TextInput transferQuestionTitle =
-                TextInput.create(TRANSFER_QUESTION_TITLE_ID, "Title", TextInputStyle.SHORT)
-                    .setMaxLength(70)
-                    .setMinLength(8)
-                    .setValue(createTitle(originalMessage))
-                    .build();
-
-        TextInput transferQuestionInput = TextInput
-            .create(TRANSFER_QUESTION_INPUT_ID, "Transfer question menu", TextInputStyle.PARAGRAPH)
-            .setValue(originalMessage)
-            .setRequiredRange(3, 2000)
+        TextInput modalTitle = TextInput.create(MODAL_TITLE_ID, "Title", TextInputStyle.SHORT)
+            .setMaxLength(70)
+            .setMinLength(4)
+            .setPlaceholder("describe question in short")
```

Comment:
start of sentence, capital. `Describe` and please write proper english. `Describe the question in short`